Source/WebKit/UIProcess/API/glib/WebKitError.cpp

```diff
@@ -35,7 +35,7 @@ using namespace WebCore;
  */
 GQuark webkit_network_error_quark()
 {
-    return g_quark_from_static_string(reinterpret_cast<const char*>(API::Error::webKitNetworkErrorDomain().span8().data()));
+    return g_quark_from_static_string(API::Error::webKitNetworkErrorDomain().utf8().data());
```

Comment:
All of these changes introduce a use after free because `g_quark_from_static_string()` does not take ownership of the string. I will change them all to `g_quark_from_string()` before landing.

Reply:
Use a different PR for this change or rename the bug, please